packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/Shapeshift/FirstStep/validationMessages.js

```diff
@@ -16,6 +16,12 @@ const Wrapper = styled.div`
   > * { margin-left: 2px; }
 `
 
+export const AboveRegulationLimitMessage = (regulationLimit) => (
+  <Text size='12px' weight={300} color='error'>
+    <FormattedMessage id='modals.exchange.abovelimitmessage' defaultMessage='Please enter an amount less than the limit.' />
```

Comment:
It would be nice if this was actionable and displayed the regulation limit, 'Please enter an amount less than the **limit'**' like its sibling 'MaximumAmountMessage'